src/elm/Api/Graphql.elm

```diff
@@ -18,6 +19,14 @@ withAuthToken authToken =
             identity
 
 
+query2 : String -> Maybe String -> SelectionSet a RootQuery -> (RemoteData (Graphql.Http.Error a) a -> msg) -> Cmd msg
+query2 endpoint authToken query_ toMsg =
```

Comment:
maybe I'm a little bit decontextualized here but since you added the auth token to loggedIn, maybe we can just keep sending `Shared` to access `endpoints`?

Reply:
Yes, we definitely can! I don't think there are any immediate tradeoffs between one or the other, just that sending `Shared` might mean a tiny little less refactoring while transitioning from `query` to `query2`, so I guess that's better :sweat_smile: